scripts/replace-build-env.js

```diff
@@ -9,21 +10,20 @@ const buildEnv = require('./build-env')
 
 const buildDirectory = process.argv[2]
 
-console.log(`Replace BuildEnv in '${buildDirectory}' with:`)
-console.log(JSON.stringify(buildEnv, null, 2))
+printLog(`Replacing BuildEnv in '${buildDirectory}' with:`, JSON.stringify(buildEnv, null, 2))
 
 try {
   const results = replace.sync({
     files: `${buildDirectory}/**/*.js`,
     from: Object.keys(buildEnv).map((entry) => `<<< ${entry} >>>`),
     to: Object.values(buildEnv),
   })
-  console.log(
+  printLog(
     'Changed files:',
     results.filter((entry) => entry.hasChanged).map((entry) => entry.file)
   )
   process.exit(0)
 } catch (error) {
-  console.error('Error occurred:', error)
+  printError('\nStacktrace:\n', error)
   process.exit(1)
```

Comment:
```suggestion
  logAndExit(error)
```

Reply:
I've changed the design of `replace-build-env.js` to be like the other scripts